tests/mongodb/test_mongo_wrapper.py

```diff
@@ -1,8 +1,8 @@
-
 import pytest
 from edgePy.data_import.mongodb.mongo_wrapper import MongoWrapper
 from edgePy.data_import.mongodb.mongo_wrapper import MongoInserter
 from edgePy.data_import.mongodb.mongo_wrapper import MongoUpdater
+from edgePy.util import LOG as log  # type: ignore
```

Comment:
Sorry, comment late to the party. I'm not sure you want to type ignore here! You should only type ignore the first `logzero` import.

Reply:
Gotcha!